prometheus-mixin/Makefile

```diff
@@ -0,0 +1,56 @@
+# Prometheus Mixin Makefile
+# Heavily copied from upstream project kubenetes-mixin
+
+# TODO: Find out why official prom images segfault
+# PROMETHEUS_IMAGE := prom/prometheus:v2.1.0
+PROMETHEUS_IMAGE := registry.gitlab.com/kskewes/docker-promtool:latest
+
+JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
```

Comment:
until `jsonnetfmt` learns to read its config from a `.jsonnetfmtrc` file in the project directory, I'm against using non-standard formatting; it just makes life harder for those people who have jsonnet autoformatters setup in their editor of choice.

Reply:
Sure. Looks like these settings have been propagating around the various mixin projects, see: [prometheus mixin](https://github.com/prometheus/prometheus/blob/master/documentation/prometheus-mixin/Makefile#L1).